draft-ietf-webpush-protocol.xml

```diff
@@ -574,24 +607,100 @@ DATA         [stream 4] +END_STREAM
   iChYuI3jMzt3ir20P8r_jgRR-dSuN182x7iB
 ]]></artwork>
       </figure>
-
       <t>
-        In response to this request, the push service MUST generate a server push
-        for all push messages that have not yet been delivered.  In addition,
-        the push service SHOULD return link references to the push and receipt
-        subscribe resources.
+        A user agent can also request the contents of the push message subscription
+        resource immediately by including a <xref target="RFC7240">Prefer header
+        field</xref> with a "wait" parameter set to "0".
       </t>
       <t>
-        A user agent can request the contents of the push message subscription
-        resource immediately by including a <xref target="RFC7240">Prefer header
-        field</xref> with a "wait" parameter set to "0".
+        In response to this request, the push service MUST generate a server push
+        for all push messages that have not yet been delivered.
       </t>
       <t>
         A 204 (No Content) status code with no associated server pushes
         indicates that no messages are presently available.  This could be
         because push messages have expired.
       </t>
+      <section anchor="monitor-set" title="Receiving Push Messages for a Subscription Set">

```

Comment:
I initially attempted that approach, but found that it lacked a certain clarity. I thought it would be a future source of issues with reviewers.